modules/settings/settings.php

```diff
@@ -260,10 +222,12 @@ public function helper_settings_validate_and_save() {
 			return false;
 		}
 
+		// phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized -- wp_verify_nonce() is already doing the necessary work
```

Comment:
This really shouldn't be needed but given this whole code is going to be simplified, it's fine for now

Reply:
For this one, you can wrap `$_POST['_wpnonce']` in `sanitize_key()` to get rid of this error. I found [this discussion about the wp_nonce sniff](https://github.com/WordPress/WordPress-Coding-Standards/issues/869#issuecomment-669119020) with a few well-known names.

Reply:
Good find - incredibly weird that it's needed but I understand why from that issue 👍🏾